jest.config.base.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     'tsx',
     'mts',
     'mtsx',
+    'cjs',
```

Comment:
`ts-api-tools` formats its output files as `.js` with ES modules. But it also includes an additional set of `.cjs` files for CJS users. This allows Jest to consider `.cjs` files (which it didn't before) - and look at them _before_ `.js` files.